### ISSUE TITLE
Could you provide the training time and GPU memory requirement?

### ISSUE THREAD

**Issue by liujiaheng** [bcmi/Causal-VidQA#3]

**Comment by liujiaheng** [comment]
the benchmark on the validation dataset?

**Comment by Jiangtong-Li** [comment]
The training time is less than 1 day, and the memory used is about 10G-20G(depend on model and batch size).
The results on validation set will be release soon.
